src/RegularExpression.js

```diff
@@ -2,8 +2,11 @@ import { GraphQLScalarType } from 'graphql';
 import { GraphQLError } from 'graphql/error';
 import { Kind } from 'graphql/language';
 
-function RegularExpression(name, regex) {
+function RegularExpression(name, regex, options = {}) {
   const REGEX = new RegExp(regex);
+  const errorMessage = options.errorMessage
```

Comment:
You may want to guard against `null` being passed here to stop an error from throwing when `options.errorMessage` is executed.

If you want to go a step further, but maybe overkill you could do a type check to ensure its an object. May be overkill 🤷‍♂️

Reply:
```
  const errorMessage = options.errorMessage
    ? options.errorMessage
    : (r, v) => `Value does not match the regular expression ${r}: ${v}`;
```

Though that could be simplified to:

```
  const errorMessage = options.errorMessage || (r, v) => `Value does not match the regular expression ${r}: ${v}`;
```